cardano-cli/src/Cardano/CLI/Shelley/Run/Transaction.hs

```diff
@@ -643,12 +644,9 @@ runTxBuild
     socketPath era (AnyConsensusModeParams cModeParams) networkId mScriptValidity
     inputsAndMaybeScriptWits readOnlyRefIns txinsc mReturnCollateral mTotCollateral txouts
     (TxOutChangeAddress changeAddr) valuesWithScriptWits mLowerBound mUpperBound
-    certsAndMaybeScriptWits withdrawals reqSigners txAuxScripts txMetadata mpparams
+    certsAndMaybeScriptWits withdrawals reqSigners txAuxScripts txMetadata
     mUpdatePropF mOverrideWits outputOptions = do
 
-  liftIO $ forM_ mpparams $ \_ ->
-    printWarning "'--protocol-params-file' for 'transaction build' is deprecated"
```

Comment:
`mpparams` is never used except for this warning, so don't pass it in the first place.